### ISSUE TITLE
Add descriptions on top of each page

### ISSUE THREAD

**Issue by NoamGaash** [hasadna/open-bus-map-search#109]
Our users doesn't understand the data shown on different pages.
We will need some text contributions describing what's going on.

The text will need approval - we have volunteers with legal understanding, but it could take more time than other contribution. Yet, this contribution is vital.

ways to contribute - 
1. write the text here in the comments section (Hebrew/English)
2. translate a text in the comment section
3. add text to the code by making a pull request. the text can be inside a `<section>` tag or inside a `<Tooltip>` component

thank you all!

**Comment by ArkadiK94** [comment]
A short welcoming description for the dashboard page, using ChatGPT ( If it is ok, I will continue for the other pages as well.)

English:
"Welcome to our platform dedicated to providing accurate bus schedules and real-time departure updates, all to ensure a more reliable and convenient bus service. Users can contribute by reporting issues like late, missed, or non-arriving buses, fostering a community-driven effort towards a better bus travel experience for all."

Hebrew:
"ברוכים הבאים לפלטפורמה שלנו שמטרתה ( החלפתי מ"המקודשת") לספק לוחות זמנים מדויקים של האוטובוסים ועדכוני יציאה בזמן אמת, הכל כדי להבטיח שירות אוטובוסים אמין ונוח יותר. המשתמשים יכולים לתרום על ידי דיווח על בעיות כמו איחורים, נסיעות שלא עצרו או שלא התקיימו, תוך יצירת מאמץ משותף שמטרתו לשפר את חוויית הנסיעה באוטובוס עבור כולם."

![Screenshot_20231106_160021_ChatGPT.jpg](https://github.com/hasadna/open-bus-map-search/assets/76536506/2b1beaa1-b11e-468a-9cca-0bf75d7b2b0a)

**Comment by NoamGaash** [comment]
This message is very generic, I'm not sure it describe what's seen on this page

**Comment by ArkadiK94** [comment]
In my opinion, we should add a welcome page / section in the dashboard so users will know the general purpose of the site.  Also, I will do something more specific for what we see in the dashboard. Is it ok that I use chatgpt for that?

**Comment by NoamGaash** [comment]
Sure, my only concern is the text quality. I want users to understand what they're seeing.
Have you seen the "about" page?

**Comment by ArkadiK94** [comment]
I thought about better approach to solve this issue. Let's make a drive word file in which every one could suggest changes to the texts. I can write the first texts and people could correct them /write better formats. 🤔

**Comment by NoamGaash** [comment]
Feel free to edit
https://docs.google.com/document/d/1jCXtb4T1uNpI8D23p-nAogorb8YVg_dsx0DKZJXj160/edit?usp=sharing

**Comment by ArkadiK94** [comment]
@NoamGaash By mistake, I changed the permissions of all users to be able only to comment before I added myself a permission to edit. Could you add only for me the permission to edit?

**Comment by NoamGaash** [comment]
The document? Anyone can contribute. That's on purpose

**Comment by NoamGaash** [comment]
The document? Anyone can contribute. That's on purpose

**Comment by ArkadiK94** [comment]
@NoamGaash Hmm, I would like to share this file in the discord and also it is here in the github. So it is open for scammers to change the text as they wish. I prefer that only few people could edit it and the others will have the permission to add a comment/suggestion. 

With that said, If you still want that everyone could edit this file. Please, change back the permissions in the file.

**Comment by NoamGaash** [comment]
@ArkadiK94 right now, anyone with link can comment.
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/f629343b-a6a9-4929-8d91-995272758001)
[It's a common practice](https://docs.google.com/document/d/1AK9agx6wz88YBI5lIsw5FCVwZmxlPNJFIJPgK693JpM/edit)

**Comment by ArkadiK94** [comment]
Yup, I did it. Could you add me that I will also be able to edit?

**Comment by shootermv** [comment]
We have a descriptions at almost all pages now...
@NoamGaash is ther more things to be done in here?

**Comment by NoamGaash** [comment]
@shootermv it's not about having description, it's more about what's in the description... phrases like "תפקוד תחבורה ציבורית לפי פרמטרים שונים" doesn't really say anything.

**Comment by shootermv** [comment]
> @shootermv it's not about having description, it's more about what's in the description... phrases like "תפקוד תחבורה ציבורית לפי פרמטרים שונים" doesn't really say anything.

personally i think it is now better than before (when there was no explanations)
but sure there is certainly a place for improvement... 
so, now it is more - `make a descriptions of the pages more explanatory` task (which requires some other skills than react knowledge), am i right?

**Comment by NoamGaash** [comment]
That's true,
The issue isn't react. it's about providing information to the user

**Comment by itai-rozen** [comment]
ive added some hebrew & english descriptions in the attached google docs. let me know what you think:)

**Comment by NoamGaash** [comment]
@itai-rozen looks great, let's hope someone will implement it :)

**Comment by itai-rozen** [comment]
For sure i will, i wanted to hear some more opinions.
Gonna also change headers in navbar to match headers in pages

בתאריך יום ו׳, 2 בפבר׳ 2024, 20:18, מאת Noam Gaash ‏<
***@***.***>:

**Comment by NoamGaash** [comment]
@itai-rozen I believe in incremental improvements - those texts will (most probably) change in the future. Yet, that's a huge  step toward better user experience, and implementing that will be  great

**Comment by NoamGaash** [comment]
I think this issue can be closed as the scope of it is too wide.
We can open separated issues for specific headings that can be improved.
Thanks everyone for this healthy discussion
